warn/warn_bazel.go

```diff
@@ -87,7 +88,7 @@ func globalVariableUsageCheck(f *build.File, category, global, alternative strin
 		start, end := ident.Span()
 		findings = append(findings,
 			makeFinding(f, start, end, category,
-				"Global variable \""+global+"\" is deprecated in favor of \""+alternative+"\". Please rename it.", true, nil))
+				fmt.Sprintf(`Global variable "%s" is deprecated in favor of "%s". Please rename it.`, global, alternative), true, nil))
```

Comment:
alternate option -- allow makeFinding to take a format-string and a var-args list, do the fmt.Sprintf in there.

Reply:
Thanks for the idea, I'll consider doing this when I refactor the warnings, currently I need to pass too many arguments to `makeFinding`, it may be confusing that some of them are for the string formatting and some are not.